app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const {handleServerErrors} = require('./controllers/error-handling')
+const {getReviews} = require('./controllers/forum-controllers')
+
+
+app.get('/api/reviews', getReviews)
+
```

Comment:
It would be good to add a path here to handle a general 404 - this can be done using '/*/ in app.js - have a look into this and see if it can implemented 😄